packages/reactotron-app/App/Stores/DevTrackerStore.js

```diff
@@ -0,0 +1,77 @@
+import { computed, observable, action, asMap } from "mobx"
+
+const TIME_NAME = 'devTrackerTimeConnected'
+const ERROR_NAME = 'devTrackerErrorsCount'
+
+class DevTracker {
+    @observable storedTime = 0
+    @observable currentErrors = 0
+
+    @observable sessionStarted = null
+
+    constructor() {
+        const currentDevTime = localStorage.getItem(TIME_NAME)
+        const currentErrors = localStorage.getItem(ERROR_NAME)
+
+        if (typeof currentDevTime === 'undefined' || currentDevTime === null) {
+            localStorage.setItem(TIME_NAME, 0)
+        } else {
+            this.storedTime = parseInt(currentDevTime, 10)
+        }
+
+        if (typeof currentErrors === 'undefined' || currentErrors === null) {
+            localStorage.setItem(ERROR_NAME, 0)
+        } else {
+            this.currentErrors = parseInt(currentErrors, 10)
+        }
+    }
+
+    @computed
+    get connectionTime() {
+        let currentSessionTime = 0
+
+        if (this.sessionStarted) {
+            const currentTime = new Date()
+            currentSessionTime = (currentTime.getTime() - this.sessionStarted.getTime()) / 1000
+        }
+
+        return this.storedTime + Math.floor(currentSessionTime)
```

Comment:
This exists because we have the time of all previous sessions but the time of the current session is not in there. So we add them together.